www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -484,5 +497,93 @@
+          <strict-origin-when-cross-origin>Strict Origin When Cross Origin</strict-origin-when-cross-origin>
+          <origin-when-cross-origin>Origin When Cross Origin</origin-when-cross-origin>
+          <unsafe-url>Unsafe URL</unsafe-url>
+        </OptionValues>
+        <Required>N</Required>
+      </referrer>
+      <xssprotection type="OptionField">
+        <multiple>N</multiple>
+        <OptionValues>
+          <val1 value="1; mode=block">Block</val1>
+          <val2 value="0">Off</val2>
+          <val3 value="1">On</val3>
+        </OptionValues>
+        <Required>N</Required>
+      </xssprotection>
+      <content_type_options type="BooleanField">
+        <Required>Y</Required>
+      </content_type_options>
+      <strict_transport_security_time type="IntegerField">
+        <Required>N</Required>
+      </strict_transport_security_time>
+      <strict_transport_security_include_subdomains type="BooleanField">
+        <Required>Y</Required>
+        <default>1</default>
+      </strict_transport_security_include_subdomains>
+      <hpkp_keys type="CSVListField">
+        <Required>N</Required>
+        <mask>/[a-z0-9\+\/=]+(,[a-z0-9\+\/=]+)*/i</mask>
+      </hpkp_keys>
+      <hpkp_report_only type="BooleanField">
+        <Required>Y</Required>
+      </hpkp_report_only>
+      <hpkp_time type="IntegerField">
+        <Required>N</Required>
+      </hpkp_time>
+      <hpkp_include_subdomains type="BooleanField">
+        <Required>Y</Required>
+      </hpkp_include_subdomains>
+      <enable_csp type="BooleanField">
+        <Required>Y</Required>
+      </enable_csp>
+      <csp_report_only type="BooleanField">
+        <Required>Y</Required>
+        <default>0</default>
+      </csp_report_only>
+      <csp_default_src_enabled type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_enabled>
+      <csp_default_src_data_urls type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_data_urls>
+      <csp_default_src_http_urls type="CSVListField">
+        <Required>Y</Required>
+      </csp_default_src_http_urls>
+      <csp_default_src_inline type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_inline>
+      <csp_default_src_eval type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_eval>
+      <csp_default_src_self type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_self>
+      <csp_default_src_blob type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_blob>
+      <csp_default_src_mediastream type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_mediastream>
+      <csp_default_src_filesystem type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_filesystem>
+      <csp_default_src_none type="BooleanField">
+        <Required>Y</Required>
+      </csp_default_src_none>
+    </security_header>
```

Comment:
@fichtner an idea how to not copy that from https://github.com/opnsense/plugins/pull/773/files#diff-8fda16d8ce523643673f4b91b34aea3fR557 to here for all of https://github.com/opnsense/plugins/pull/773/files#diff-ca27b344b01813de50f2239c2091dfb3R24 ?